core/themes/basis/css/component/hero.css

```diff
@@ -94,3 +94,21 @@
     font-size: 2.625em;
   }
 }
+
+/**
+ * Some changes to fix problems with extra white space between blocks in Basis.
+ * Is there a better location?
+ **/
+
+.update-1-30 .block {
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.update-1-30 .container .l-flexible-row {
```

Comment:
I don't think we should include row-specific CSS in the hero.css file. This probably belongs in `layout.css` (once we're happy with the changes).

Reply:
Yes, I was planning to get advice on where to put this.